commit 50b020cdb7f72077e16133f1d88c9359cf415a53
Author: Ricardo Koller <ricarkol@google.com>
Date:   Mon Nov 8 18:38:55 2021 -0800

    KVM: selftests: aarch64: Add vgic_irq to test userspace IRQ injection
    
    Add a new KVM selftest, vgic_irq, for testing userspace IRQ injection.  This
    particular test injects an SPI using KVM_IRQ_LINE on GICv3 and verifies
    that the IRQ is handled in the guest. The next commits will add more
    types of IRQs and different modes.
    
    Signed-off-by: Ricardo Koller <ricarkol@google.com>
    Acked-by: Andrew Jones <drjones@redhat.com>
    Signed-off-by: Marc Zyngier <maz@kernel.org>
    Link: https://lore.kernel.org/r/20211109023906.1091208-7-ricarkol@google.com

diff --git a/tools/testing/selftests/kvm/.gitignore b/tools/testing/selftests/kvm/.gitignore
index 3763105029fb..f75d4c433fc3 100644
--- a/tools/testing/selftests/kvm/.gitignore
+++ b/tools/testing/selftests/kvm/.gitignore
@@ -4,6 +4,7 @@
 /aarch64/get-reg-list
 /aarch64/psci_cpu_on_test
 /aarch64/vgic_init
+/aarch64/vgic_irq
 /s390x/memop
 /s390x/resets
 /s390x/sync_regs_test